config/routes.rb

```diff
@@ -52,6 +53,8 @@
     end
   end
 
+  mount GrapeAPI::RootAPI => "/"
```

Comment:
app/api/ is the grape default. If you want to setup differently we can try move it to lib and see if that works.